GitUI/CommandsDialogs/RepoHosting/CreatePullRequestForm.cs

```diff
@@ -104,11 +106,17 @@ private void _pullReqTargetsCB_SelectedIndexChanged(object sender, EventArgs e)
             _remoteBranchesCB.Items.Clear();
             _remoteBranchesCB.Text = _strLoading.Text;
 
-            AsyncLoader.DoAsync(
-                () => _currentHostedRemote.GetHostedRepository().Branches,
-                branches =>
+            // TODO merge with nearly identical code below
```

Comment:
💡 File follow up issues instead of adding comments

Reply:
Extracted method and removed TODO.